lexicons/app/bsky/feed/defs.json

```diff
@@ -44,7 +44,8 @@
         "like": { "type": "string", "format": "at-uri" },
         "threadMuted": { "type": "boolean" },
         "replyDisabled": { "type": "boolean" },
-        "embeddingDisabled": { "type": "boolean" }
+        "embeddingDisabled": { "type": "boolean" },
+        "pinned": { "type": "boolean" }
```

Comment:
Just curious, does this affect the presentation of a post elsewhere in the app (other than a user's own author feed) when viewing their own pinned post?

Reply:
We use it to show "Pin post" or "Unpin post" in the context menu